compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__correct_pipe_arity_error_location.snap

```diff
@@ -6,4 +6,4 @@ error: Incorrect arity
   ┌─ /src/one/two.gleam:2:18
   │
 2 │ fn main() { 1 |> x() }
-  │                  ^^^ Expected 2 arguments, got 0
+  │                  ^^^ Expected 2 arguments, got 1
```

Comment:
Both would be correct but I believe that the new one is going to be a bit more useful, my hunch is that the case where people meant `left |> right(..args)` to be `right(left, ..args)` are more common than `right(..args)(left)`